features/storage/blockdevice/MBRBlockDevice.h

```diff
@@ -66,7 +68,9 @@ class MBRBlockDevice : public BlockDevice {
      *
      *  @param bd       Block device to partition
      *  @param part     Partition to use, 1-4
-     *  @param type     8-bit partition type to identify partition contents
+     *  @param type     The 8-bit partition type is to specify the actual filesystem to use/format in this partition.
+     *                  It can be found from filesystem documentation or refer to
+     *                  https://en.wikipedia.org/wiki/Partition_type#List_of_partition_IDs
```

Comment:
```suggestion
     *  @param type     8-bit partition type to specify the filesystem to use in this partition.
     *                  It can be found in the filesystem documentation or refer to
     *                  https://en.wikipedia.org/wiki/Partition_type#List_of_partition_IDs
```